mrp_unbuild_cust_alu_analytics_maintenance/views/mrp_unbuild_views.xml

```diff
@@ -0,0 +1,28 @@
+<odoo>
+    <data>
+        <record id="mrp_unbuild_form_incidence_ids_maintenance_equipment" model="ir.ui.view">
+            <field name="name">mrp.unbuild.form (maintenance_equipment) (mrp_unbuild_maintenance)</field>
+            <field name="model">mrp.unbuild</field>
+            <field name="inherit_id" ref="mrp_unbuild_cust_alu_analytics.mrp_unbuild_form_incidence_ids_analytics"/>
+            <field name="groups_id" eval="[(4,ref('maintenance.group_equipment_manager'))]"/>
```

Comment:
This is too restricted. According to 

https://github.com/odoo/odoo/blob/13.0/addons/maintenance/security/ir.model.access.csv

every user can access a maintenance equipment, so we can remove such access without any problem.

Reply:
Removed